tutorials/xr/setting_up_xr.rst

```diff
@@ -91,7 +92,37 @@ Next we need to add a script to our root node. Add the following code into this
             # Change our main viewport to output to the HMD
             get_viewport().use_xr = true
         else:
-            print("OpenXR not initialised, please check if your headset is connected")
+            print("OpenXR not initialized, please check if your headset is connected")
+
+  .. code-tab:: csharp
+    using Godot;
+    using System;
```

Comment:
From what I can see in other documentation examples, we usually keep a blank line at the beginning. Also, since the `System` namespace doesn't seem to be needed, we tend to omit it in the documentation.

```suggestion
  .. code-tab:: csharp

    using Godot;
```